lib/mix/tasks/sentry.install.ex

```diff
@@ -0,0 +1,188 @@
+defmodule Mix.Tasks.Sentry.Install.Docs do
+  @moduledoc false
+
+  def short_doc do
+    "Installs sentry. Requires igniter to be installed."
+  end
+
+  def example do
+    "mix sentry.install --dsn <your_dsn>"
+  end
+
+  def long_doc do
+    """
+    #{short_doc()}
+
+    ## Example
+
+    ```bash
+    #{example()}
+    ```
+
+    ## Options
+
+    * `--dsn` - Your sentry `dsn`
+    """
+  end
+end
+
+if Code.ensure_loaded?(Igniter) do
+  defmodule Mix.Tasks.Sentry.Install do
+    @shortdoc "#{__MODULE__.Docs.short_doc()}"
+
+    @moduledoc __MODULE__.Docs.long_doc()
+
+    use Igniter.Mix.Task
+
+    @impl Igniter.Mix.Task
+    def info(_argv, _composing_task) do
+      %Igniter.Mix.Task.Info{
+        group: :sentry,
+        adds_deps: [{:jason, "~> 1.2"}, {:hackney, "~> 1.8"}],
+        example: __MODULE__.Docs.example(),
+        schema: [dsn: :string],
+        # Default values for the options in the `schema`
+        defaults: [dsn: "<your_dsn>"]
+      }
+    end
+
+    @impl Igniter.Mix.Task
+    def igniter(igniter) do
+      app_name = Igniter.Project.Application.app_name(igniter)
+
+      mix_env_code =
+        quote do
+          Mix.env()
+        end
+
+      cwd_code =
+        quote do
+          [File.cwd!()]
+        end
+
+      # Do your work here and return an updated igniter
+      igniter
+      |> Igniter.Project.Config.configure(
+        "prod.exs",
+        app_name,
+        [:dsn],
+        igniter.args.options[:dsn]
+      )
+      |> Igniter.Project.Config.configure(
+        "prod.exs",
+        app_name,
+        [:environment_name],
+        {:code, mix_env_code}
+      )
+      |> Igniter.Project.Config.configure(
+        "prod.exs",
+        app_name,
+        [:enable_source_code_context],
+        true
+      )
+      |> Igniter.Project.Config.configure(
+        "prod.exs",
+        app_name,
+        [:root_source_code_paths],
+        {:code, cwd_code}
```

Comment:
Can we trim the `quote`d vars above a bit?

```suggestion
        {:code, quote(do: [File.cwd!()]}
```